module/input/Camera/data/config/Cameras/Left.yaml

```diff
@@ -6,6 +6,16 @@ settings:
   Height: 1024
   OffsetX: 0
   OffsetY: 0
-  ExposureAuto: Continuous
-  GainAuto: Continuous
-  BalanceWhiteAuto: Continuous
+  ExposureAuto: Continuous # Options: Off, Once, Continuous
+  GainAuto: Continuous # Options: Off, Once, Continuous
+  Gain: 15 # Range: 0.00 - 18.01'
+  GammaEnable: true # Set to false to adjust gamma. Default = true
+  Gamma: 0.50
+  BalanceWhiteAuto: Once # Options: Off, Once, Continuous
+  BalanceRatioSelector: Red # Options: Red or Blue. Controls the balance ratio of the selected color relative to green. Used for white balancing.
+  BalanceRatio: 2.56
+  AutoExposureEVCompensation: 0 # Range from -3(darker) to 3(lighter) with a step of 1/3. Default = 0.
+  AcquisitionFrameRateEnable: true
+  AcquisitionFrameRate: 20.00
```

Comment:
Might be good to have a quick note here about this helping with flickering

Reply:
What kind of flickering were you getting?

Reply:
Was kind of pulsating from the fluros in the lab. One of our photography minded members suggested adjusting the frame rate to suit the grid frequency. I played around with it a bit and this seemed like a sweet spot.